R/write_output.R

```diff
@@ -0,0 +1,329 @@
+#' Write model outputs to specified directories
+#'
+#' Processes the model fit, extracts samples and quantiles,
+#' and writes them to the appropriate directories.
+#'
+#' @param fit An EpiNow2 fit object with posterior estimates.
+#' @param output_dir String. The base output directory path.
+#' @param samples A data.table as returned by [process_samples()]
+#' @param summaries A data.table as returned by [process_quantiles()]
+#' @param job_id String. The identifier for the job.
+#' @param task_id String. The identifier for the task.
+#' @param metadata List. Additional metadata to be included in the output.
```

Comment:
I don't think documentation of config-derived parameters should live here. I think they should be provided in the config and documented in the readme or associated documentation as you suggested in #7. I'd like to do that work in #7 rather than here.

Is there some tweak to the documentation here that I can make that would make that solution work for you?

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/issues/65